modules/network-observability-dns-tracking.adoc

```diff
@@ -13,32 +13,37 @@ CPU and memory usage increases are observed in the eBPF agent when this feature
 ====
 .Procedure
 . In the web console, navigate to *Operators* -> *Installed Operators*.
-. Under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*.
+. Under the *Provided APIs* heading for *Network Observability*, select *Flow Collector*.
 . Select *cluster* then select the *YAML* tab.
 . Configure the `FlowCollector` custom resource. A sample configuration is as follows:
 +
 [id="network-observability-flowcollector-configuring-dns_{context}"]
 .Configure `FlowCollector` for DNS tracking
 [source, yaml]
 ----
-apiVersion: flows.netobserv.io/v1alpha1
+apiVersion: flows.netobserv.io/v1beta2
 kind: FlowCollector
 metadata:
   name: cluster
 spec:
   namespace: netobserv
-  deploymentModel: DIRECT
+  deploymentModel: Direct
   agent:
-    type: EBPF
+    type: eBPF
     ebpf:
       features:
        - DNSTracking           <1>
-      privileged: true         <2>
+      sampling: 1              <2>
 ----
 <1> You can set the `spec.agent.ebpf.features` parameter list to enable DNS tracking of each network flow in the web console.
-<2> Note that the `spec.agent.ebpf.privileged` specification value must be `true` for DNS tracking to be enabled.
+<2> You can set `sampling` to a value of `1` for more accurate metrics.
 
 . When you refresh the *Network Traffic* page, there are new DNS representations you can choose to view in the *Overview* and *Traffic Flow* views and new filters you can apply.
 .. Select new DNS choices in *Manage panels* to display graphical visualizations and DNS metrics in the *Overview*.
 .. Select new choices in *Manage columns* to add DNS columns to the *Traffic Flows* view.
-.. Filter on specific DNS metrics, such as *DNS Id*, *DNS Latency* and *DNS Response Code*, and see more information from the side panel.
\ No newline at end of file
+.. Filter on specific DNS metrics, such as *DNS Id*, *DNS Error* *DNS Latency* and *DNS Response Code*, and see more information from the side panel. The *DNS Latency* and *DNS Response Code* columns are shown by default.
+
+[NOTE]
+===
+TCP handshake packets do not have DNS headers. TCP protocol flows without DNS headers are shown in the traffic flow data with *DNS Latency*, *ID*, and *Response code* values of "n/a". You can filter out flow data to view only flows that have DNS headers using the *Common* filter "DNSError" equal to "0". 
+===
```

Comment:
```suggestion
[NOTE]
====
TCP handshake packets do not have DNS headers. TCP protocol flows without DNS headers are shown in the traffic flow data with *DNS Latency*, *ID*, and *Response code* values of "n/a". You can filter out flow data to view only flows that have DNS headers using the *Common* filter "DNSError" equal to "0". 
====
```
Need an extra `=` to fix the formatting here
<img width="772" alt="Screenshot 2024-01-12 at 12 46 17 PM" src="https://github.com/openshift/openshift-docs/assets/93731574/f6dc408b-b343-4a97-8fea-e7cfd129972f">